crates/ironrdp-pdu/src/rdp/client_info.rs

```diff
@@ -597,6 +597,12 @@ bitflags! {
     }
 }
 
+impl Default for PerformanceFlags {
+    fn default() -> Self {
+        Self::DISABLE_FULLWINDOWDRAG | Self::DISABLE_MENUANIMATIONS | Self::ENABLE_FONT_SMOOTHING
+    }
+}
```

Comment:
thought: I think the default flags should not live in the PDU crate as this is a concern which is a bit more far-reaching.